InternxtDesktop/Views/Backup/FolderSelectorView.swift

```diff
@@ -0,0 +1,89 @@
+//
+//  FolderSelectorView.swift
+//  InternxtDesktop
+//
+//  Created by Richard Ascanio on 12/28/23.
+//
+
+import SwiftUI
+
+struct FolderSelectorView: View {
+
+    let closeWindow: () -> Void
+    @State private var isBackupButtonEnabled = false
+    @State private var filenames: [String] = []
+    @State private var urls: [String] = []
+    @State private var selectedIndex: Int?
+
+    var body: some View {
+        VStack(spacing: 12) {
+
+            HStack {
+                AppText("BACKUP_SETTINGS_BACKUP_FOLDERS")
+                    .font(.LGMedium)
+                    .foregroundColor(.Gray100)
+
+                Spacer()
+
+                Text("BACKUP_SETTINGS_\("\(urls.count)")_FOLDERS")
+                    .font(.BaseRegular)
+                    .foregroundColor(.Gray50)
+            }
+
+            WidgetFolderList(folders: $filenames, selectedIndex: $selectedIndex)
+
+            HStack {
+                HStack(spacing: 8) {
+                    AppButton(icon: .Plus, title: "", onClick: {
+                        let panel = NSOpenPanel()
+                        panel.allowsMultipleSelection = false
+                        panel.canChooseDirectories = true
+                        panel.canChooseFiles = false
+                        if panel.runModal() == .OK {
+                            if let url = panel.url {
+                                let filename = url.lastPathComponent
```

Comment:
if we are choosing folders, user foldername instead of filename, this is really confusing otherwise